renovate.json

```diff
@@ -2,15 +2,18 @@
 	"extends": [ "config:base" ],
 	"lockFileMaintenance": { "enabled": true },
 	"ignoreDeps": [ "husky" ],
-	"schedule": [ "before 3am on wednesday" ],
 	"composer": {
 		"enabled": false
 	},
-  "labels": ["type: dependencies", "skip-changelog"],
+	"labels": ["type: dependencies", "skip-changelog"],
 	"packageRules": [
 		{
 			"packageNames": [ "automattic/jetpack-autoloader" ],
 			"rangeStrategy": "bump"
-		}
-	]
+		},
+    {
+      "updateTypes": ["minor", "patch", "pin", "digest"],
+      "automerge": true
+    }
+  ]	]
```

Comment:
There is an errant extra bracket here and the indenting is off.

Reply:
Woah, good spotting. I think I need to change my json colour scheme - it seems to hide anything erroneous like this. 🤦 

Now fixed :)